explorer/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# build environment
+FROM node:16-alpine as react-build
+WORKDIR /app
+COPY ./package.json ./package-lock.json ./
+RUN npm install
+COPY . ./
+RUN npm run build
+
+# server environment
+FROM nginx:alpine
+COPY ./nginx.conf /etc/nginx/conf.d/configfile.template
```

Comment:
Would it be possible to do this without bringing in the nginx.conf template? For example, in this tutorial  https://medium.com/@shakyShane/lets-talk-about-docker-artifacts-27454560384f

Reply:
It can be done, but this will lose the user's control over monitoring PORT. For the practice of Docker containerization, it may not be so flexible.